matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java

```diff
@@ -358,7 +380,8 @@ private JSONObject buildMatchesJSONArray(List<PatientMatch> matches)
         final JSONObject matchesJson = this.patientMatchExport.toJSON(matches);
         return new JSONObject()
             .put(RESULTS_LABEL, matchesJson.optJSONArray("matches"))
-            .put(RETURNED_SIZE_LABEL, matchesJson.optJSONArray("matches").length());
+            .put(RETURNED_SIZE_LABEL, matchesJson.optJSONArray("matches").length())
+            .put("dateGenerated", DATE_TIME_SDF.format(new Date()));
```

Comment:
also why do we add `dateGenerated` here, but `parameters` on line 367 - makes sense to have all new changes in one place?